k9mail/src/test/java/org/openintents/openpgp/util/ShadowOpenPgpAsyncTask.java

```diff
@@ -0,0 +1,25 @@
+package org.openintents.openpgp.util;
+
+
+import java.util.concurrent.Executor;
+
+import android.content.Intent;
+import android.os.AsyncTask;
+
+import org.robolectric.annotation.Implementation;
+import org.robolectric.annotation.Implements;
+import org.robolectric.annotation.RealObject;
+import org.robolectric.shadows.ShadowAsyncTask;
+
+
+@Implements(OpenPgpApi.OpenPgpAsyncTask.class)
+public class ShadowOpenPgpAsyncTask extends ShadowAsyncTask<Void, Integer, Intent> {
```

Comment:
I've never looked closely into Robolectric's shadow classes. What does this do and why is it necessary?